docs/sources/send-data/promtail/cloud/gcp/_index.md

```diff
@@ -273,7 +273,7 @@ We also exclude specific HTTP load balancer logs based on payload and status cod
 
 ```
 $ gcloud logging sinks create cloud-logs pubsub.googleapis.com/projects/my-project/topics/cloud-logs \
---log-filter='resource.type=("gcs_bucket OR k8s_cluster OR service_account OR iam_role OR api OR audited_resource OR http_load_balancer")' \
+--log-filter='resource.type=("gcs_bucket OR Kubernetes_cluster OR service_account OR iam_role OR api OR audited_resource OR http_load_balancer")' \
```

Comment:
Here `k8s_cluster` is the name of a field so it should not be changed.

Reply:
Whoops, thanks for catching that.  Reverted.